### PR TITLE
feat(starters): add mhadaily/gatsby-starter-typescript-power-blog

### PR DESCRIPTION
**Pull request by mhadaily**
Starter Repo: https://github.com/mhadaily/gatsby-starter-typescript-power-blog
Starter Demo : https://gatsby-starter-typescript-power-blog.majidhajian.com

### CODE REVIEW

File: docs/starters.yml [hunk 1]
```diff
@@ -1061,3 +1061,26 @@
     - Antd component library pre-installed
     - Uses gatsby-plugin-antd for modular imports
     - Customize the theme of Antd with `modifyVars`
+- url: https://gatsby-starter-typescript-power-blog.majidhajian.com/
+  repo: https://github.com/mhadaily/gatsby-starter-typescript-power-blog
+  description: Minimal Personal Blog with Gatsby and Typescript
+  tags:
+    - PWA
+    - Blog
+    - Personal website
```
Comment (LekoArts) [inline docs/starters.yml]: I think `Blog` is enough. `Personal website` doesn't give any extra value

File: docs/starters.yml [hunk 2]
```diff
@@ -1061,3 +1061,26 @@
...
+    - TypeScript
+    - Progressive web app
```
Comment (LekoArts) [inline docs/starters.yml]: Please omit this tag as you're already using `PWA`

File: docs/starters.yml [hunk 3]
```diff
@@ -1061,3 +1061,26 @@
...
+    - Markdown
+  features:
+    - Mobile-First approach in development.
```
Comment (LekoArts) [inline docs/starters.yml]: Nitpick: You switch between using/not using a dot at the end of the sentence

Comment (mhadaily) [reply docs/starters.yml]: done

File: docs/starters.yml (hunk 2 above)
Comment (mhadaily) [reply docs/starters.yml]: done

File: docs/starters.yml (hunk 1 above)
Comment (mhadaily) [reply docs/starters.yml]: done

**Pull request merged**